doc/UserGuide.md

```diff
@@ -34,7 +34,7 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/BLOCK, STREET, UNIT, POSTAL_CODE [t/TAG]...` 
```

Comment:
I think perhaps I should put [p]a/BLOCK [COMMA] STREET [COMMA] UNIT [COMMA] POSTAL_CODE?